{"content": {"suites": {"0": {"status": {"total_pass": 1, "total_skip": 2, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 1, "total_error": 0}, "tests": {"0": {"status": "SKIP", "message": "('G:\\\\SQA\\\\Automation Practice\\\\E_Commerce\\\\test_ecommerce.py', 29, 'Skipped: Already created..')\n", "test_name": "test_click_singUp", "rerun": "0"}, "1": {"status": "SKIP", "message": "('G:\\\\SQA\\\\Automation Practice\\\\E_Commerce\\\\test_ecommerce.py', 52, 'Skipped: Already created..')\n", "test_name": "test_register_info", "rerun": "0"}, "2": {"status": "PASS", "message": "", "test_name": "test_login", "rerun": "0"}, "3": {"status": "FAIL", "message": "   selenium.common.exceptions.WebDriverException: Message: Reached error page: about:neterror?e=networkProtocolError&u=https%3A//automationexercise.com/products&c=UTF-8&d=Firefox%20has%20experienced%20a%20network%20protocol%20violation%20that%20cannot%20be%20repaired.\n   Stacktrace:\n   RemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\n   WebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:180:5\n   UnknownError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:507:5\n   checkReadyState@chrome://remote/content/marionette/navigate.sys.mjs:61:24\n   onNavigation@chrome://remote/content/marionette/navigate.sys.mjs:329:39\n   emit@resource://gre/modules/EventEmitter.sys.mjs:154:20\n   receiveMessage@chrome://remote/content/marionette/actors/MarionetteEventsParent.sys.mjs:35:25\n", "test_name": "test_add_to_cart", "rerun": "0"}}, "suite_name": "test_ecommerce.py"}}}, "date": "February 14, 2023", "start_time": 1676364694.2026134, "total_suite": 1, "status": "FAIL", "status_list": {"pass": "1", "fail": "1", "skip": "2", "error": "0", "xpass": "0", "xfail": "0", "rerun": "0"}, "total_tests": "4"}